modules/libzip/1.10.1/patches/add-build-file.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/BUILD b/BUILD
+new file mode 100644
+index 00000000..991cb183
+--- a/BUILD
++++ b/BUILD
+@@ -0,0 +1,52 @@
++load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
++load("@rules_license//rules:license.bzl", "license")
++
++package(
++    default_applicable_licenses = [":license"],
++    default_visibility = ["//visibility:public"],
++)
++
++license(
++    name = "license",
++    package_name = "libzip",
++)
++
++filegroup(
++    name = "all_srcs",
++    srcs = glob(["**"]),
++)
++
++cmake(
```

Comment:
Is it possible to write a cc_library for this target?

Reply:
It is possible butI don't have the time for that right now unfortunately. This cmake target is what we already use as the BUILD file to load as a non_module_dep now.  I'm right now just trying to add the ones we have already to BCR.

https://github.com/kythe/kythe/blob/master/third_party/libzip.BUILD shows a way to build this using cc_library but that involves using a custom "expand_template" bzl rule.